realsense2_camera/src/ros_utils.cpp

```diff
@@ -80,6 +80,22 @@ std::string create_graph_resource_name(const std::string &original_name)
     return fixed_name;
 }
 
+bool string_to_rs2_format(std::string str , rs2_format* format)
```

Comment:
I would rather work with 
rs2_format string_to_rs2_format(std::string str)
that returns a RS2_FORMAY_ANY if not found..